lib/node_modules/@stdlib/string/next-grapheme-cluster-break/test/test.js

```diff
@@ -0,0 +1,151 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var nextGraphemeClusterBreak = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof nextGraphemeClusterBreak, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function throws an error if the first argument is not a string primitive', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		-5,
+		3.14,
+		-1.0,
+		NaN,
+		true,
+		false,
+		null,
+		void 0,
+		[ 'beep', 'boop' ],
+		[ 1, 2, 3 ],
+		{},
+		function noop() {}
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			nextGraphemeClusterBreak( value, 0 );
+		};
+	}
+});
+
+tape( 'the function throws an error if the second argument is not a nonnegative integer', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		-5,
+		'bar',
+		3.14,
+		-1.0,
+		NaN,
+		null,
+		void 0,
+		[ 'beep', 'boop' ],
+		[ 1, 2, 3 ],
+		{},
+		function noop() {}
+	];
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			nextGraphemeClusterBreak( 'foo', value );
+		};
+	}
+});
+
+tape( 'the function throws an error if a provided position is greater than the string length', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		[ 'bar', 3 ],
+		[ 'string', 7 ]
+	];
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), RangeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			nextGraphemeClusterBreak( value[ 0 ], value[ 1 ] );
+		};
+	}
+});
+
+tape( 'the arity of the function is 2', function test( t ) {
```

Comment:
This test can be removed in its entirety. We typically don't test arity, unless arity is important for downstream users, and this varies case-by-case.